sklearn/mixture/_gaussian_mixture.py

```diff
@@ -514,6 +514,15 @@ class GaussianMixture(BaseMixture):
 
     Read more in the :ref:`User Guide <gmm>`.
 
+    For examples on different methods of initialization, refer to:
+    :ref:`sphx_glr_auto_examples_mixture_plot_gmm_init.py`
+
+    For examples on model selection with Gassian Mixture, refer to:
+    :ref:`sphx_glr_auto_examples_mixture_plot_gmm_selection.py`
```

Comment:
"For a detailed example of how the `n_components` and `covariance_type` parameters influence the model's performance, see <example_link>."